### PR TITLE
CC-2137: downgrade dependencies to pre-poetry versions

### PR DESCRIPTION
**Pull request by jonholdsworth**
## Jira Ticket

[CC-2137 NZSL: move signbank to Poetry](https://ackama.atlassian.net/browse/CC-2137)

## Changes

- This restores the exact package versions from the original `requirements.txt`

### CODE REVIEW

**Comment by jonholdsworth** [comment]
The pinned Certifi version for NZSL Signbank is failing OSV.
I can pin it at the GHSA approved version in pyproject.toml
Or would you prefer I do it separately in a Dependabot PR? That would still require unpinning it.

Update: We are not bothering for now. Will not prevent CI deployment.

File: pyproject.toml [hunk 1]
```diff
@@ -9,28 +9,29 @@ homepage = "https://signbank.nzsl.nz/"
...
+django-guardian="2.2.0"
+django-notifications-hq="1.6.0"
+dj-database-url="0.5.0"
+boto3="1.22.8"
+django-storages="1.12.1"
+psycopg2="2.8.6"
+certifi="2023.7.22"
+sentry-sdk="1.14.0"
+gunicorn = "22.0.0"
+whitenoise="6.2.0"
```
Comment (G-Rath) [inline pyproject.toml]: issue: can you restore a space around the `=`s?

Comment (jonholdsworth) [reply pyproject.toml]: Sure!

Comment (jonholdsworth) [reply pyproject.toml]: Done.

**Review by G-Rath — APPROVED** [review]
squash when merging

**Pull request merged**